pkg/customizations/oscap/stage_options.go

```diff
@@ -64,3 +64,38 @@ func CreateTailoringStageOptions(oscapConfig *blueprint.OpenSCAPCustomization, d
 		},
 	)
 }
+
+func CreateRemediationStageOptions(
+	oscapConfig *blueprint.OpenSCAPCustomization,
+	isOSTree bool,
+	d distro.Distro,
+) (*osbuild.OscapRemediationStageOptions, []*fsnode.Directory, error) {
+	if oscapConfig == nil {
+		return nil, nil, nil
+	}
+
+	if isOSTree {
+		return nil, nil, fmt.Errorf("unexpected oscap options for ostree image type")
+	}
```

Comment:
This is IMO not the place to check if the image type is ostree or not. This is not the job if a function which generates an osbuild stage options. Instead, this should be checked on a higher-lervel in the distro implementation and we should return an error there in case one wants to use BP customizations with an image type that does not support them.